src/styles/main.scss

```
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./variables.scss";
+@import "./header.scss";
+@import "./catalog.scss";
+@import "./reset.scss";
+@import "./star.scss";
```

Import downloads the whole css file and applies all of its content to the final css file. It is important here to watch the element order.
For example here you will use reset.scss after catalog and header styles has been used.
```suggestion
@import "./variables.scss";
@import "./reset.scss";
@import "./header.scss";
@import "./catalog.scss";
@import "./star.scss";
```